src/main/java/vazkii/botania/common/item/lens/LensMine.java

```diff
@@ -62,13 +64,24 @@ public boolean collideBurst(IManaBurst burst, EntityThrowable entity, MovingObje
 					if(ConfigHandler.blockBreakParticles)
 						entity.worldObj.playAuxSFX(2001, x, y, z, Block.getIdFromBlock(block) + (meta << 12));
 
-					int dropX = warp ? coords.posX : x;
-					int dropY = warp ? coords.posY : y;
-					int dropZ = warp ? coords.posZ : z;
-
-					for(ItemStack stack_ : items)
-						world.spawnEntityInWorld(new EntityItem(world, dropX + 0.5, dropY + 0.5, dropZ + 0.5, stack_));
-
+					if(warp && coords.posY == -1) {
+						if(burst instanceof EntityManaBurst) {
+							EntityPlayer sourcePlayer = ((EntityManaBurst) burst).getSourcePlayer();
```

Comment:
Ideally this function would be part of the IManaBurst interface, but I'm refraining from modifying anything in the API package.